src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -585,20 +587,6 @@ private void transferAlignmentInfoToPairedRead(final SAMRecord firstUnaligned, f
     }
 
     /**
-     * Updates the MD and NM tags if present.
-     * @param rec
-     */
-    private void calculateMdAndNmTags(final SAMRecord rec) {
-        // update NM and MD
-        final boolean calculateNm = rec.getIntegerAttribute(ReservedTagConstants.NM) != null;
-        final boolean calculateMd = rec.getStringAttribute(SAMTag.MD.name()) != null;
-        if (calculateNm || calculateMd) {
-            final byte[] referenceBases = this.refSeq.get(this.refSeq.getSequenceDictionary().getSequenceIndex(rec.getReferenceName())).getBases();
-            SequenceUtil.calculateMdAndNmTags(rec, referenceBases, calculateNm, calculateMd);
```

Comment:
I agree. especially important for booleans. but all params could benefit. not sure why java don't have it too....